### ISSUE TITLE
Load previously trained model.

### ISSUE THREAD

**Issue by arongdari** [arongdari/deep-distant-supervision#3]
When the model tries to load the previously trained model, the current code is not working correctly :cold_sweat:

```python
saver = tf.train.Saver()
if tf.train.latest_checkpoint(save_path):
    saver.restore(sess, save_path)
    logger.info("Last Session Restored")
```
with the error message:

> NotFoundError (see above for traceback): Unsuccessful TensorSliceReader constructor: Failed to find any matching files for checkpoints/batch_size=48/beta1=0.9/beta2=0.999/bidirectional=True/dataset=data/nyt/dropout=True/epsilon=1e-08/h=False/helpfull=False/is_train=True/keep_prob=0.5/learning_rate=0.001/len_sentence=70/load_prev=True/log_level=INFO/max_batch_sentences=1500/max_position=123/num_epoch=10/num_hidden=256/num_relation=53/pos_dim=5/pretrained_w2v=True/print_gap=50/reg_weight=0.0001/save_gap=100/sent_attn=True/w2v_path=data/vec.npy/word_attn=True/
	 [[Node: model/save/RestoreV2 = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_model/save/Const_0_0, model/save/RestoreV2/tensor_names, model/save/RestoreV2/shape_and_slices)]]
